service/controller/v26/resource/endpoint/current.go

```diff
@@ -72,6 +72,13 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 			return nil, microerror.Mask(err)
 		}
 
+		if k8sEndpoints != nil {
```

Comment:
Can `k8sEndpoints` be `nil` when `errors.IsNotFound` is handled with `return`?

Reply:
You are correct, I was a bit too paranoid because we allow the service to be nil.